packages/rainbowkit/src/components/ConnectOptions/ConnectDetails.tsx

```diff
@@ -429,6 +429,7 @@ export function InstructionDetail({
               height="48"
               minWidth="48"
               overflow="hidden"
+              position="relative"
```

Comment:
Is this an accidental change?

Reply:
not accidental, this fixed a safari bug i noticed with the instruction icons:
<img width="381" alt="Screen Shot 2022-06-29 at 1 22 21 AM" src="https://user-images.githubusercontent.com/16931094/176358211-f07716b4-3ac2-4470-8049-f755d1866659.png">